### PR TITLE
Change code example in 2.5

### PR DESCRIPTION
**Pull request by oafischer**
Changes copied from: https://github.com/stolostron/backlog/issues/24240

### CODE REVIEW

File: clusters/upgrade_cluster_discon.adoc [hunk 1]
```diff
@@ -223,31 +223,34 @@ metadata:
...
+                kind: ImageContentSourcePolicy
                 metadata:
-                  name: sample-nginx-pod
-                  namespace: default
-                status:
-                  phase: Running
-          remediationAction: inform
-          severity: low
-  remediationAction: enforce
+                  name: your-local-mirror
```
Comment (dockerymick) [inline clusters/upgrade_cluster_discon.adoc]: update `name:` value to `<your-local-mirror-name>`

This suggestion comes from my previous [PR](https://github.com/stolostron/rhacm-docs/pull/3701/files#diff-7f308fede3321b98bda56851a0b3d758720d0f8a322eb11ead1775cb9229407aR287)

File: clusters/upgrade_cluster_discon.adoc [hunk 2]
```diff
@@ -223,31 +223,34 @@ metadata:
...
+                spec:
+                  repositoryDigestMirrors:
+                    - mirrors:
+                        - <path-to-local-mirror>
```
Comment (dockerymick) [inline clusters/upgrade_cluster_discon.adoc]: Update `- mirrors:` value from `<path-to-local-mirror>` to `<your-registry>`

This suggestion comes from my previous [PR](https://github.com/stolostron/rhacm-docs/pull/3701/files#diff-7f308fede3321b98bda56851a0b3d758720d0f8a322eb11ead1775cb9229407aR291)

**Review by dockerymick — CHANGES_REQUESTED** [review]
Made a suggestion based on a previous PR

File: clusters/upgrade_cluster_discon.adoc (hunk 2 above)
Comment (oafischer) [reply clusters/upgrade_cluster_discon.adoc]: Made both changes, thanks for pointing those out!

**Review by dockerymick — APPROVED** [review]
Thanks for accepting my suggestion, this looks good to me

**Pull request merged**